outlines/blaze.md

```diff
@@ -0,0 +1,56 @@
+# Blaze
+
+1. Introduction to Blaze -- the tracker-backed handlebars like templating syntax
+  1. Example of spacebars syntax, data context + helpers
```

Comment:
I think in this guide, we should go into detail about data context and how it relates to "Views" - we gloss over this in the intro tutorial completely.